Source/PeleC_setup.cpp

```diff
@@ -225,9 +225,20 @@ PeleC::variableSetUp ()
 	cnt += NumSpec;
     }
 
+#ifdef SOOT_MODEL
```

Comment:
Is it necessary to use ifdefs for the `SOOT_MODEL`? Can they be replaced by `if` conditions? This would help in the test matrix of exectuables.

Reply:
You mean eliminate the preprocessor directives for soot altogether and change them to C++ checks? Or just change the preprocessor from ifdef to if?

Reply:
yes, sorry I mean c++ checks

Reply:
I could get rid of it once I clean up the soot code and remove all of the managed memory. At the moment, it create less problems to have it separate.